test/io/test_cli.py

```diff
@@ -289,3 +289,78 @@ def test_jwt_aud_config_set_to_invalid_uri(defaultenv):
     with pytest.raises(PostgrestError):
         dump = cli(["--dump-config"], env=env).split("\n")
         assert "jwt-aud should be a string or a valid URI" in dump
+
+
+def test_cli_ready_flag_success(defaultenv):
+    "PostgREST ready flag succeeds when ready"
+
+    env = {
+        **defaultenv,
+        "PGRST_ADMIN_SERVER_PORT": "3001",
+    }
+
+    # Run postgrest process
+    command = [POSTGREST_BIN]
+    env["HPCTIXFILE"] = hpctixfile()
+    pgrst_process = subprocess.Popen(command, env=env)
```

Comment:
Since these lines are repeated in the tests, can we refactor them into a reusable function? Seems like it could fit in `test/io/postgrest.py`.